packages/core/test/validation-profiles.test.ts

```
@@ -0,0 +1,125 @@
+    expect(out.seed).toBe(1);
+    expect(out.network).toBeDefined();
+    expect(out.network!.latencies!.length).toBeGreaterThan(0);
+    expect(out.network!.latencies!.some((l) => l.delayMs === 1500)).toBe(true);
+  });
+
+  it('throws unknown_profile when profile name is not registered', () => {
+    expect.assertions(3);
+    try {
+      prepareChaosConfig({ profile: 'nope' });
+    } catch (e) {
+      expect(e).toBeInstanceOf(ChaosConfigError);
```

ESLint

[error] Avoid calling `expect` inside conditional statements

(vitest/no-conditional-expect)